### DISCUSSION TITLE
How to set prometheus persistence?

### DISCUSSION THREAD

**Discussion by qingfeng777** [prometheus-operator/kube-prometheus#1656] [Q&A] [answered]
if pod of prometheus restarted, the data is missing. 
how to set PV to prometheus operator? my version is v0.7.0

thans very much

**Comment by simonpasquier** [comment, marked as answer]
https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/storage.md should provide the instructions. If you're really using Prometheus operator v0.7.0, I would strongly recommend that you upgrade because this is a very old release.

**Reply by qingfeng777** [reply]
thanks very much.
 about the Version,  our kubernetes's version is 1.20, so for us, the choice is v0.7.0 or v0.8.0 . looks nearly the same ？
